.travis.yml

```diff
@@ -20,7 +20,7 @@ jobs:
   include:
     # Basic Checks
     - stage: primary
-      env: NOXSESSION=lint-3.5
+      env: NOXSESSION=lint-3.8
```

Comment:
Maybe take out this change since we are planning to move away from Travis ?

Reply:
That is why the PR is labelled `WIP`. It is still functionally incomplete.